cartography/intel/aws/route53.py

```diff
@@ -58,14 +58,19 @@ def link_aws_resources(neo4j_session: neo4j.Session, update_tag: int) -> None:
 def load_a_records(neo4j_session: neo4j.Session, records: List[Dict], update_tag: int) -> None:
     ingest_records = """
     UNWIND {records} as record
-    MERGE (a:DNSRecord:AWSDNSRecord{id: record.id})
-    ON CREATE SET a.firstseen = timestamp(), a.name = record.name, a.type = record.type
-    SET a.lastupdated = {update_tag}, a.value = record.value
-    WITH a,record
-    MATCH (zone:AWSDNSZone{zoneid: record.zoneid})
-    MERGE (a)-[r:MEMBER_OF_DNS_ZONE]->(zone)
-    ON CREATE SET r.firstseen = timestamp()
-    SET r.lastupdated = {update_tag}
+        MERGE (a:DNSRecord:AWSDNSRecord{id: record.id})
```

Comment:
These queries are the same but I've added whitespace to make them easier to read.